actions.yaml

```diff
@@ -50,3 +50,8 @@ rebalance:
       description: Only generate the partition rebalance proposals and estimated result, without executing
       type: boolean
       default: true
+    brokerid:
+      description: Broker ID newly added to the cluster or to be removed
+      type: integer
+      minimum: 0
+  required: [mode, dryrun]
```

Comment:
**todo:** `dryrun` shouldn't be required, as it's the default behavior if ran without `dryrun=false`